ci/4-setup-cluster-env

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# NOTE: The script should be executed as a part of ci/run!
+# NOTE: As this script exports environment variables, it should be run with
+#       `source ci/4-setup-cluster-env` in order for them to be made available.
+
+# SCRIPT SUMMARY
+# --------------
+# 1. Export docker environment variables
+# 2. Export development and testing environment variables
+
+# Set shell options
+# -e : Exit immediately if a command exits with a non-zero status.
+# -u : Treat unset variables as an error when substituting.
+# -x : Print commands and their arguments as they are executed.
+set -ux
+
+# 1.
+if set +u && [[ -z "${CICD}" ]] && set -u; then
+  # LOCAL
+  eval $(minikube docker-env)
```

Comment:
I don't know if it will hurt to run on Travis, but I don't think we need to do it as there is a difference between running locally and on Travis. Locally we use a VM while setting up minikube: `--vm-driver=virtualbox` while on Travis we use: `--vm-driver=none`.

Reply:
Ok, if on travis minikube has access to the same dockerd that chartpress uses to build images then this isn't needed. Thanks for teaching me something about the inner workings of minikube :)

Reply:
Thats it! Thanks for teaching me about docker haha lets add more comments!